version/version.go

```diff
@@ -6,7 +6,7 @@ const (
 	// VersionMajor is for an API incompatible changes
 	VersionMajor = 1
 	// VersionMinor is for functionality in a backwards-compatible manner
-	VersionMinor = 7
+	VersionMinor = 8
```

Comment:
BTW if we are at all serious about semantic versioning, this needs to be 2.0.0.

Reply:
Did API change?

Reply:
Yes.

Reply:
Well lets move to 2.0.0 when we have mirroring.